src/Targets/Microsoft.Managed.DesignTime.targets

```diff
@@ -265,4 +265,7 @@
 
   </Target>
 
+  <!-- This target collects all Analyzers in the project. -->
+  <Target Name="CollectAnalyzersDesignTime" DependsOnTargets="CompileDesignTime" Returns="@(Analyzer)" />
```

Comment:
>@(Analyzer) [](start = 90, length = 11)

Is there any problem if Analyzer contains duplicates? I recall that CollectPackageReference (above) hit an exception with duplicates (discussed [https://github.com/dotnet/project-system/issues/1290#issuecomment-275295124](https://github.com/dotnet/project-system/issues/1290#issuecomment-275295124)) but we decided this was a CPS bug